layouts/_default/volunteer.html

```diff
@@ -6,7 +6,7 @@
         <div class="page-section page-section-grid-priority">
           <div
             class="page-section-image page-section-grid-secondary"
-            style='background-image: url("/images/general/volunteer.jpg")'
+            style='background-image: url("/images/general/volunteer_2.jpg")'
```

Comment:
Do we have a higher resolution version or similar picture?

![image](https://github.com/user-attachments/assets/3fc9e786-03bf-40f1-96f3-3f788c67b5d4)

Also, since the bulk of this page is a slim center aligned text box, should we have the "hero section", then the image in the vertical hierarchy?